test/integration/members.test.js

```diff
@@ -51,10 +40,9 @@ describe("Members", function () {
     });
 
     afterEach(async function () {
-      await addUser(userWithoutRolesObject);
-      await addUser(userWithRolesObjectWithoutArchivedProperty);
-      await addUser(userWithRolesObjectWithArchivedTrue);
-      await addUser(userWithRolesObjectWithArchivedFalse);
+      await cleanDb();
+      await addUser(userArchivedRoleFalse);
+      await addUser(userAlreadyArchived);
```

Comment:
Should this be in 'before'?

Reply:
Yes. Resolved by https://github.com/Real-Dev-Squad/website-backend/issues/680